target/linux/generic/files/drivers/net/phy/rtl8367b.c

```diff
@@ -785,9 +817,14 @@ static int rtl8367b_extif_set_mode(struct rtl8366_smi *smi, int id,
 		return -EINVAL;
 	}
 
-	REG_RMW(smi, RTL8367B_DIS_REG,
-		RTL8367B_DIS_RGMII_MASK << RTL8367B_DIS_RGMII_SHIFT(id),
-		mode << RTL8367B_DIS_RGMII_SHIFT(id));
+	if (id < 2)
```

Comment:
Please don't use magic numbers without compiler variables. What does id < 2 mean and can we be sure that it doesn't break other chips' support?